web/config/defaultSettings.ts

```diff
@@ -42,3 +43,20 @@ export default {
     test: string;
   };
 };
+
+const { dev, test } = defaultSettings.serveUrlMap;
+const throwPromptError = (message: TemplateStringsArray) => {
+  throw new Error(
+    `Please set '${message[0]}' in 'web/.env' file. Guide: https://apisix.apache.org/docs/dashboard/develop/#web`,
+  );
+};
+
+if (REACT_APP_ENV === 'test' && !test) {
+  throwPromptError`SERVER_URL_TEST`;
```

Comment:
LGTM, but should we use the most generic function call syntax? like `throwPromptError("SERVER_URL_TEST");`, although it still works, it may be confusing for those unfamiliar with it.

Reply:
Although we don't seem to have used it much in this project yet, it is actually relatively common, for example in emotion, styled components, graphql...